logger.go

```diff
@@ -89,8 +89,15 @@ func (log *logger) With(fields ...Field) Logger {
 }
 
 func (log *logger) Check(lvl Level, msg string) *CheckedMessage {
-	if !(lvl >= log.Level()) {
-		return nil
+	switch lvl {
+	case PanicLevel, FatalLevel:
+		// Panic and Fatal should always cause a panic/exit, even if the level
+		// is disabled.
+		break
+	default:
+		if lvl < log.Level() {
```

Comment:
This is just some minor code cleanup, it does not change the behaviour of the `if`.

I think it's easier to read `if a < b` vs `if !(a >= b)`. The same change is also made in zwrap/sample.go.